tests/integration-tests/tests/tests-test.js

```diff
@@ -478,6 +479,33 @@ describe('the tests api', function() {
             });
         });
     });
+    describe('Delete tests', () => {
+        it('try to delete test with scheduled jobs and fail', async () => {
+            const requestBody = simpleTest.test;
+            const createTestResponse = await testsRequestSender.createTest(requestBody, validHeaders);
+            const testId = createTestResponse.body.id;
+
+            const jobsBody = {
```

Comment:
Hi @zer0link 
according to this test, it's impossible to delete a test which is not scheduled
Blocking delete test should be only for cron tests